docs/content/en/integration/openid-connect/firezone/index.md

```diff
@@ -0,0 +1,91 @@
+---
+title: "Firezone"
+description: "Integrating Firezone with the Authelia OpenID Connect Provider."
+lead: ""
+date: 2023-03-25T13:07:02+10:00
+draft: false
+images: []
+menu:
+  integration:
+    parent: "openid-connect"
+weight: 620
+toc: true
+community: true
+---
+
+## Tested Versions
+
+* [Authelia]
+  * [v4.37.5](https://github.com/authelia/authelia/releases/tag/v4.37.5)
+* [Firezone]
+  * [0.7.25](https://github.com/firezone/firezone/releases/tag/0.7.25)
+
+## Before You Begin
+
+{{% oidc-common %}}
+
+### Assumptions
+
+This example makes the following assumptions:
+
+* __Application Root URL:__ `https://firezone.example.com`
+* __Authelia Root URL:__ `https://auth.example.com`
+* __Client ID:__ `firezone`
+* __Client Secret:__ `insecure_secret`
+* __Authentication Name (Gitea):__ `authelia`:
+    * This option determines the redirect URI in the format of
+      `https://firezone.example.com/user/oauth2/<Authentication Name>/callback`.
+      This means if you change this value you need to update the redirect URI.
+
+## Configuration
+
+### Application
+
+To configure [Firezone] to utilize Authelia as an [OpenID Connect 1.0] Provider:
+
+1. Sing in with an admin user
+2. Visit Settings -> Security
+3. In the "Single Sign-On" section, click on the "Add OpenID Connect Provider" button
+4. Configure:
+   1. Config ID: `authelia`
+   2. Label: `Authelia`
+   3. Scope: `openid email profile`
+   4. Client ID: `firezone`
+   5. Client secret: `insecure_secret`
+   6. Discovery Document URI: `https://auth.example.com/.well-known/openid-configuration`
+   7. Redirect URI (optional): `https://firezone.example.com/auth/oidc/authelia/callback/`
```

Comment:
```suggestion
   7. Redirect URI (optional): `https://firezone.example.com/auth/oidc/authelia/callback`
```

Reply:
This is the change I'd like tested. While their docs include it, it shouldn't be required.